x-pack/plugins/security_solution/public/common/components/toolbar/bulk_actions/use_bulk_alert_tags_items.tsx

```diff
@@ -26,10 +45,23 @@ export const useBulkAlertTagsItems = ({ refetch }: UseBulkAlertTagsItemsProps) =
     },
   ];
 
-  const alertTagsPanels = [
+  const TitleContent = useMemo(
+    () => (
+      <EuiFlexGroup alignItems="center" gutterSize="s" responsive={false}>
+        <EuiFlexItem grow={false}>{i18n.ALERT_TAGS_CONTEXT_MENU_ITEM_TITLE}</EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiIconTip content={i18n.ALERT_TAGS_CONTEXT_MENU_ITEM_TOOLTIP_INFO} position="right" />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    ),
+    []
+  );
+
+  const alertTagsPanels: UseBulkAlertTagsPanel[] = [
```

Comment:
Since `alertTagsPanels` is being being used as the the dependency in `useMemo` here, can we consider memoizing `alertTagsPanels` where `renderContent` can be Memoized Component?